pkg/vpcmodel/debugOutput.go

```diff
@@ -20,7 +20,13 @@ func (t *DebugOutputFormatter) WriteOutput(c1, c2 *VPCConfig,
 		out = subnetsConn.String()
 	case SingleSubnet:
 		out = c1.GetConnectivityOutputPerEachSubnetSeparately()
+	case SubnetsDiff, EndpointsDiff:
```

Comment:
What do you mean "update headerOfAnalyzedVPC ?

Indeed, excluding AllEndpoints debug is identical to text. Wouldn't it be better to say DEBUG is not supported in these modes it is actually identical to text?

Reply:
Currently the call to `headerOfAnalyzedVPC ` here is with empty string for vpc2 name, although this is not the case for `diff`. 
I'm ok with supporting `debug` output only for the case where it is different from `text`.

Reply:
both fixes inserted

Reply:
is it intentional that `headerOfAnalyzedVPC` gets empty string for vpc2 name now? (because diff not supported)?
if so, should not consider in `SingleAnalysisOutput` the field `VPC2Name`.
for the unsupported usecases I would print a message that the output is given in text format. (currently empty output is returned).

Reply:
1. Yes, it gets empty string for DEBUG since diff is not supported for DEBUG. Diff is supported for other formats, hence the field` VPC2Name` in `SingleAnalysisOutput`
2. I changed the parser so that in `DEBUG` only all_endpoints is possible. So no need to print anything in these cases

Reply:
ok, as for (1) -- I meant that in the function `func (t *DebugOutputFormatter) WriteOutput` , there is no need to compute `v2Name`  and assign it within `SingleAnalysisOutput.VPC2Name`,  since we assume this function is not relevant for `diff` usecases.

Reply:
right. fixed